### ISSUE TITLE
Logging learning process

### ISSUE THREAD

**Issue by nzw0301** [nzw0301/supervised_fasttext.pytorch#2]
Keras's histroy format is good:

- train loss
- val loss
- val accuracy
- test loss
- test accuracy
